spec/components/schemas/Payments/RequestSources/PaymentRequestQPaySource.yaml

```
@@ -0,0 +1,28 @@
+###
+# Warning: this file was auto generated from OpenAPI specs using 'npm run sync-generated-specs'. Do not manually edit. 
+###
+type: object
+description: 'QPay Source'
+allOf:
+  -
+    $ref: '#/components/schemas/PaymentRequestSource'
+  -
+    type: object
+    required:
+      - quantity
+      - description
+    properties:
+      quantity:
+        minimum: 1
+        type: integer
+        description: 'A numeric value greater than zero represents the quantity of purchased Item. The value is used for display purposes only and does not affect the total amount of the payment. The minimum allowed value is 1.'
+      description:
+        type: string
+        description: 'Alphanumeric string containing a narrative description of the payment order. Note: The maximum allowed length of this property is 255 characters after UTF-8 URL encoding.'
```

I'm not sure how narrative
```suggestion
        description: 'Alphanumeric string containing a description of the payment order. Note: The maximum allowed length of this property is 255 characters after UTF-8 URL encoding.'
```
I'm not sure how `narrative` fits in this context. Is this a story about the payment order? :)

---

not really sure where it came from, I think Adam wrote those specs in the first place, but I can't tag him here since he doesn't have access to the repo:D

---

Adam confirmed it's just strange QPay wording